src/stores/useScreenConfigStore.ts

```diff
@@ -0,0 +1,26 @@
+// stores/useScreenConfigStore.js
+import { create } from 'zustand';
+
+const BASE_WIDTH = 393;
+const BASE_HEIGHT = 852;
+
+type screenConfigState = {
+  screenWidth: number;
+  screenHeight: number;
+  updateConfig: () => void;
+};
+
+export const useScreenConfigStore = create<screenConfigState>(set => ({
+  screenWidth: BASE_WIDTH,
+  screenHeight: BASE_HEIGHT,
+  updateConfig: () => {
+    const userAgent = navigator.userAgent;
+    const isMobile =
+      /Mobile|Android/i.test(userAgent) || window.innerWidth <= 768;
+    const screenWidth = isMobile ? window.innerWidth : BASE_WIDTH;
+    const screenHeight = isMobile ? window.innerHeight : BASE_HEIGHT;
+
+    set({ screenWidth });
+    set({ screenHeight });
```

Comment:
```suggestion
    set({ screenWidth, screenHeight });
```

이렇게 한 번에 쓰는 것도 동작하나용??

Reply:
앗.. 작성해 주신 코드로도 충분히 동작합니다 😂